tests/bluetooth/mesh/microbit.conf

```diff
@@ -18,9 +18,6 @@ CONFIG_BT_PHY_UPDATE=n
 # This test barely fits in RAM, don't pull in logging
 CONFIG_TEST_LOGGING_DEFAULTS=n
 
-# Use legacy LL until RAM utilization is optimized in the new LL
```

Comment:
Does this means, the RAM size requirement with split controller is worked around/acceptable?

Reply:
Apparently yes, because it builds fine